src/styles/AddItem.css

```diff
@@ -0,0 +1,34 @@
+.container {
```

Comment:
This CSS looks similar to what's in @DeeDowns PR and your PR's may clash with each other. You may want to touch base with each other to make sure that they play together well.

Reply:
Oh, thanks!